tests/redisearch.ts

```diff
@@ -51,14 +62,69 @@ describe('RediSearch Module testing', async function() {
         }])
         expect(response).to.equal('OK', 'The response of the FT.CREATE command');
     });
+
     it('search function', async () => {
         const response = await client.search(index, query)
         expect(response).to.equal(0, 'The response of the FT.SEARCH command')
     });
+    it('search function response test', async () => {
```

Comment:
Instead of adding a new test, please add the contents of this test into the test above.

Reply:
they are testing different kind of response, different kind of scenario. They should be in a separate block.